src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowUpdate.cs

```diff
@@ -30,6 +32,14 @@ public sealed class RowUpdate : RowEditBase
         private const string UpdateScript = "UPDATE {0} SET {1} {2}";
         private const string UpdateScriptMemOptimized = "UPDATE {0} WITH (SNAPSHOT) SET {1} {2}";
         private const string SelectStatement = "SELECT {0} FROM {1}";
+        private string ValidateUpdateOnlyOneRow = "DECLARE @numberOfRows int = 0;" + Environment.NewLine +
+                                                          "Select @numberOfRows = count(*) FROM {0} {1} " + Environment.NewLine +
+                                                          "IF (@numberOfRows > 1) " + Environment.NewLine +
+                                                          "Begin" + Environment.NewLine +
+                                                           " DECLARE @error NVARCHAR(100) = N'The row value(s) updated do not make the row unique or they alter multiple rows(' + CAST(@numberOfRows as varchar(10)) + ' rows)';" + Environment.NewLine +
```

Comment:
It may be hard to localize this error message?

Reply:
I fix the localization later